build/common_unix.go

```diff
@@ -18,7 +18,7 @@ import (
 
 // ReloadPlugin - kills any running instances and waits for grafana to reload the plugin
 func ReloadPlugin() error {
-	exeName, err := getExecutableName(runtime.GOOS, runtime.GOARCH, defaultPluginJSONPath)
+	exeName, err := getExecutableNameForPlugin(runtime.GOOS, runtime.GOARCH, defaultPluginJSONPath)
```

Comment:
`getExecutableName` is tagged as deprecated so I am using the new one.

@wbrowne do you see any reason to not use this method? I tested it locally and it is working correctly.

Reply:
The main difference with this is that it will only search the provided pluginJSON path. So as long as we're not dealing with multiple plugins here, then it should be okay?